engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -92,6 +98,8 @@ object SlickDataAccess {
   lazy val log = LoggerFactory.getLogger("slick")
 
   val FailureEventMaxMessageLength = 1024
+
+  private final case class ExecutionAndAux(execution: WorkflowExecution, aux: WorkflowExecutionAux)
```

Comment:
Let's call this `WorkflowExecutionAndAux`. Unfortunately the term "Execution" is overloaded and in its unqualified state usually refers to `call`s